src/index.html

```diff
@@ -458,6 +459,11 @@ <h3 class="gallery__heading">Łukasz Nowacki</h3>
     </p>
   </div>
 </footer>
+<!-- Inline for faster parsing -->
+<script>
+  document.querySelector('body').classList.toggle('with-js');
+  document.querySelector('.main-header').classList.toggle('main-header--sticky');
```

Comment:
Correct me if I'm wrong, but I suggest change `querySelector('body')` to `document.body` because it's much faster. 
[https://jsperf.com/document-body-vs-document-queryselector-body](https://jsperf.com/document-body-vs-document-queryselector-body)

`document.getElementsByClassName('.main-header')[0]` is faster than `document.querySelector('.main-header')` too.
[https://jsperf.com/ca-getelementclassname-vs-queryselector](https://jsperf.com/ca-getelementclassname-vs-queryselector)